charts/agent-control-deployment/values.yaml

```diff
@@ -103,6 +103,9 @@ config:
     # @default -- See `values.yaml`
     port: 51200
 
+  # -- List of allowed chart repository URLs. The Agent Control will only allow to deploy agents from these repositories.
+  # @default -- `[]`(Only newrelic chart repositories allowed: ["https://helm-charts.newrelic.com","https://newrelic.github.io/<>"])
```

Comment:
How do you know that these are the default values? They change based on the agent type right? This can't be enforced from here.